examples/repl/cmdline.rs

```diff
@@ -898,10 +898,15 @@ pub async fn cmdline(context: Context, line: &str, chat_id: &mut ChatId) -> Resu
                 None
             };
 
+            let time_start = std::time::SystemTime::now();
             let msglist = context.search_msgs(chat, arg1).await?;
+            let time_needed = std::time::SystemTime::now()
```

Comment:
Can be `time_start.elapsed().unwrap_or_default()`.

Reply:
There is also similar code above.

Reply:
nice! i will change that, also in the other line that stops the time.